sdk/model/src/config/model.ts

```diff
@@ -149,26 +149,36 @@ export type CfgResourceValueAsJson = {
 
 export type CfgBlobContent = {
   type: 'GetBlobContent';
+  fromBytes: number | undefined;
```

Comment:
Why not to put it into its own type (i.e. `rangeContent`) and use composition here and in all types below?